OpenSim/Simulation/SimbodyEngine/Joint.h

```diff
@@ -60,16 +60,14 @@ A Seth, M Sherman, P Eastman, S Delp; Nonlinear dynamics 62 (1), 291-303
 \code{.cpp}
     // Define a Pin joint between ground and platform.
```

Comment:
Can you add a note saying something like `The "platform" body must be added to the Model by the time you call `initSystem()`?